Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```
@@ -7420,6 +7420,20 @@ ViewTransitionsEnabled:
     WebCore:
       default: true
 
+ViewTransitionsTypeEnabled:
+  type: bool
+  category: animation
+  status: testable
+  humanReadableName: "View Transitions :active-view-transition-type() support"
+  humanReadableDescription: "Support specifying active view transition types"
```

```suggestion
  humanReadableName: "View Transition Types"
  humanReadableDescription: "Support specifying types for view transitions"
```

It's not just about the CSS selector fwiw.